docs/release-notes/release-notes-0.19.0.md

```diff
@@ -234,6 +234,9 @@ config option](https://github.com/lightningnetwork/lnd/pull/9182) and introduce
 a new option `channel-max-fee-exposure` which is unambiguous in its description.
 The underlying functionality between those two options remain the same.
 
+* [Improved user experience](https://github.com/lightningnetwork/lnd/pull/9454)
```

Comment:
Should be 0.18.5 now i guess?

Reply:
I pushed a commit to move it into 0.18.5.